books.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I suggest that you make `rentals` an `attr_accessor` as well we need to read and modify it in this case